src/components/TextInput/TextInput.vue

```diff
@@ -73,6 +73,28 @@ export default {
       default: '',
       required: true
     },
+    type: {
+      type: String,
+      default: 'text',
+      validator: function (value) {
+        return ['date', 'email', 'number', 'password', 'tel', 'text', 'url'].includes(value);
+      }
+    },
+    // Used by number inputs.
+    min: {
```

Comment:
Should these props also be in the input itself (e.g. `<input :min="min" :max="max" :pattern="pattern" />`)?

Reply:
Yes, thank you for catching that.